hack/istio/install-error-rates-demo.sh

```diff
@@ -54,12 +54,26 @@ echo NAMESPACE_ALPHA=${NAMESPACE_ALPHA}
 echo NAMESPACE_BETA=${NAMESPACE_BETA}
 echo SOURCE=${SOURCE}
 
+IS_OPENSHIFT="false"
+IS_MAISTRA="false"
+if [[ "${CLIENT_EXE}" = *"oc" ]]; then
+  IS_OPENSHIFT="true"
+  IS_MAISTRA=$([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true" || echo "false")
+fi
+
+echo "IS_OPENSHIFT=${IS_OPENSHIFT}"
+echo "IS_MAISTRA=${IS_MAISTRA}"
+
+
 # If we are to delete, remove everything and exit immediately after
 if [ "${DELETE_DEMO}" == "true" ]; then
   echo "Deleting Error Rates Demo (the envoy filters, if previously created, will remain)"
-  if [ "${CLIENT_EXE}" == "oc" ]; then
-    ${CLIENT_EXE} delete network-attachment-definition istio-cni -n ${NAMESPACE_ALPHA}
-    ${CLIENT_EXE} delete network-attachment-definition istio-cni -n ${NAMESPACE_BETA}
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    if [ "${IS_MAISTRA}" != "true" ]; then
+      $CLIENT_EXE delete network-attachment-definition istio-cni -n ${NAMESPACE_ALPHA}
+      $CLIENT_EXE delete network-attachment-definition istio-cni -n ${NAMESPACE_BETA}
+    fi
+    $CLIENT_EXE delete security-context-constraints error-rates-scc
```

Comment:
```suggestion
    $CLIENT_EXE delete scc error-rates-scc
```

Reply:
fixed in my PR here https://github.com/jshaughn/kiali/pull/5/files#diff-be5229a9ec29e499939f17823afb8a72b1dfdea8cb386d7b5ef1245f5fce1a9dR113
nothing to do other than just merge my PR.
you should rebase your PR first.